src/components/shared/menu.js

```diff
@@ -14,35 +14,25 @@ function createPopup(doc) {
     e.returnValue = false;
   }
 
-  let mask = document.querySelector("#contextmenu-mask");
-  if (!mask) {
-    mask = doc.createElement("div");
-    mask.id = "contextmenu-mask";
-    if (document.body) {
-      document.body.appendChild(mask);
+  let debuggerWindow = document.getElementsByClassName("debugger")[0];
+  if (debuggerWindow) {
+    debuggerWindow.onclick = () => {
+      if (popup) {
+        popup.hidePopup();
+      }
```

Comment:
my concern with this approach is that some click handlers will prevent propogation so this will not always work.

I think the safer thing to do would be to limit the impact of right clicking on a mask so that it doesn't show a context menu.

for instance preventDefault, stop propogation